Proto/cg.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
```

Comment:
@dsyme I really think FAKE is the library you are missing here :). It has a lot helpers for everything but is kind of massive nowadays... In fact I'm trying to push FAKE in a more 'library' like way and improve F# scripting in general. Just look [here](https://github.com/fsharp/FAKE/issues/1232)